pytest_serverless.py

```diff
@@ -226,18 +226,22 @@ def serverless():
 
 def _load_file():
     serverless_path = os.environ.get("SERVERLESS_FILE_PATH", "serverless.yml")
+    serverless_command = os.environ.get("SERVERLESS_COMMAND", "sls")
 
     is_serverless = os.path.isfile(serverless_path)
     if not is_serverless:
         raise Exception("No serverless.yml file found!")
 
-    if not which("sls"):
-        raise Exception("No sls executable found!")
+    if serverless_command != "sls" and serverless_command != "serverless":
```

Comment:
```suggestion
    if serverless_command not in ("sls", "serverless"):
```
You could slightly simplify this `if` statement

Reply:
Thanks for the suggestion!